Source/WebCore/animation/TimelineRange.h

```diff
@@ -34,11 +34,24 @@ struct SingleTimelineRange {
 
     Name name { Name::Normal };
     Length offset;
+
+    bool operator==(const SingleTimelineRange& other) const = default;
+
+    enum class StartOrEnd : bool { Start, End };
```

Comment:
I know we debated this already, but even after your explanation I probably would have named this `Type` instead of `StartOrEnd`.

Reply:
Fixed.